### ISSUE TITLE
The HTML table sizes and content dont look as expected

### ISSUE THREAD

**Issue by tristanius** [dompdf/dompdf#1304]
Hi, The HTML table sizes and content do not look as expected. 

I tried print an HTML as:

    <h1>DomPDF</h1>

    <table>
      <tbody>
        <tr>
          <td>
            test
          </td>
          <td>
            test
          </td>
        </tr>
        <tr>
          <td>
            test
          </td>
          <td>
            test
          </td>
        </tr>
      </tbody>
    </table>
In my localhost is OK, is correct, but on my ubuntu server look like this:

![captura](https://cloud.githubusercontent.com/assets/6276711/20194337/e12141c2-a75f-11e6-952f-b8a1d52ec392.PNG)

Help me please, i tried change php version 5.5, 5.6 and 7.0, reinstall modules required by DomPDF, but nothing...

**Comment by tristanius** [comment]
The HTML is
    <h1>DomPDF</h1>

    <table>
      <tbody>
        <tr>
          <td>
            test
          </td>
          <td>
            test
          </td>
        </tr>
        <tr>
          <td>
            test
          </td>
          <td>
            test
          </td>
        </tr>
      </tbody>
    </table>

**Comment by bsweeney** [comment]
Is that the extent of your HTML/CSS? It seems like maybe a font metrics issue, but your code sample doesn't indicate you're using a custom font.

**Comment by tristanius** [comment]
Yes, thats all HTML/CSS. Anyone had this problem before?

**Comment by bsweeney** [comment]
Not really. It seems like maybe the font metrics are missing/corrupted. Does dompdf have read/write ability on the configured temp, font, and font cache directories?
